ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCUI-nonprod.json

```diff
@@ -562,5 +562,27 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "MANAGE_DOCUMENTS",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-systemupdate",
+          "caseworker-civil-admin",
+          "caseworker-civil"
+        ],
+        "CRUD": "CRUD"
+      },
+      {
+        "UserRoles": [
+          "APP-SOL-SPEC-PROFILE",
```

Comment:
Can citizen defendant also view the event?

Reply:
No, it will be only managed by caseworker , so removed these chnages